examples/MagicWeather/package.json

```diff
@@ -7,7 +7,8 @@
     "ios": "react-native run-ios",
     "start": "react-native start",
     "test": "jest",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "pods": "bundle install && pod-install --quiet"
```

Comment:
Before we were also removing the `Podfile.lock` file in `MagicWeather`. Is this change intentional?

Reply:
yes, I wasn't sure why we were doing that, so I was like, ok let's remove it and see if it breaks something. I think it was just for those cases where the version changes, which I believe `--repo-update` should help with. I will add repo-update